email/html/event-update_password.ftl

```diff
@@ -2,19 +2,30 @@
 <#import "template/button.ftl" as button>
 <#import "template/layout.ftl" as layout>
 <#import "template/text.ftl" as text>
+<#import "template/textLink.ftl" as textLink>
 
 <#--  EMAIL - Update password confirmation -->
 
 <@layout.emailLayout title=msg("eventUpdatePasswordSubject")>
   <#--  ENGLISH  -->
   <@body.emailBody_en title=msg("eventUpdatePassword_title_en")>
     <@text.emailText><b>${msg("eventUpdatePassword_welcome_en")}</b></@text.emailText>
-    <@text.emailText>${msg("eventUpdatePassword_text_en")?no_esc}</@text.emailText>
+    <@text.emailText>${msg("eventUpdatePassword_text1_en")?no_esc}</@text.emailText>
+		<@text.emailText>
+			${msg("eventUpdatePassword_text2_en")?no_esc} 
+			<@textLink.emailTextLink href=properties.ohcrnInfoEmail>${properties.ohcrnInfoEmail}</@textLink.emailTextLink>
```

Comment:
For the HTML email I think we want the href to be prefixed with `mailto:` if it is an email address. Let's double-check the affected HTML emails and make sure those hrefs get the prefix, the plaintext version is good as you have it though

Reply:
added mailto - if it doesn't work there's probably an email client configuration issue on your mac (found this out the hard way)